openapi-cli/src/test/resources/generators/service/ballerina/requestBody/additional_property.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/http;
+
+listener http:Listener ep0 = new (9090, config = {host: "localhost"});
+
+service / on ep0 {
+    # Description
+    #
+    # + payload - parameter description
+    # + return - Success
+    resource function post api/HistoricoSimulacao/AddHistorico(@http:Payload record {|string...;|}|record {|HistoricoSimulacaoViewModel...;|}|record {|record {|string...;|}...;|} payload) returns http:Ok {
+    }
+    # Description
+    #
+    # + payload - parameter description
+    # + return - Success
+    resource function put api/Mailing/AddContact(@http:Payload Mailing_AddContact_body|Mailing_AddContact_body_1 payload) returns http:Ok {
+    }
+    # Description
+    #
+    # + payload - parameter description
+    # + return - Success
+    resource function post api/Mailing/AddContact(@http:Payload Mailing_AddContact_body_2|Mailing_AddContact_body_3 payload) returns http:Ok {
+    }
```

Comment:
Do we have the `types.bal` file from the generated sources.. shouldn't we ideally validate that file as well to verify the reference schemas having additional properties in these two cases?

Reply:
Yeah, I will check for testing the types.bal as well